extensions/extensions-docker/aissemble-hive/aissemble-hive-service/src/main/resources/docker/Dockerfile

```diff
@@ -1,32 +1,29 @@
-FROM openjdk:11-jre-slim
+ARG METASTORE_VERSION
+FROM apache/hive:${METASTORE_VERSION} as appsource
+
+FROM eclipse-temurin:17-jre-ubi9-minimal as final
 
 LABEL org.opencontainers.image.source = "https://github.com/boozallen/aissemble"
 
 WORKDIR /opt
 
-ENV HADOOP_VERSION=3.3.6
-ENV METASTORE_VERSION=3.0.0
-
-ENV HADOOP_HOME=/opt/hadoop-${HADOOP_VERSION}
-ENV HIVE_HOME=/opt/apache-hive-metastore-${METASTORE_VERSION}-bin
+ARG HADOOP_VERSION
+ARG METASTORE_VERSION
```

Comment:
Q: Isn't this ARG already defined above?

Reply:
This is initially redeclared due to this: https://docs.docker.com/reference/dockerfile/#understand-how-arg-and-from-interact

With that said, upon further examination, the latter ARG is unused so it can be removed.

Reply:
Interesting to note nonetheless. I wasn't aware args worked that way.